### PR TITLE
Add stats snapshot for DOS5

### PR DESCRIPTION
**Pull request by lfdebrux**
Ticket: https://trello.com/c/VOfe4Yo4/473-build-a-performance-dashboard

Digital Outcomes and Specialists 5 is coming soon, we want to be able to get stats on the applications. This PR adds the DOS5 framework to the stats snapshot job, so stats will be uploaded to Google Drive, where they will then be picked up by the script to display them as a graph.

### CODE REVIEW

File: job_definitions/stats_snapshots.yml [hunk 1]
```diff
@@ -11,7 +10,8 @@
 {#    ('g-cloud-16', '1fWI3LrnPU1cHVYo_83KWfgND-lng2ztM', False)    #}
 
 {% set frameworks = (
-    ('g-cloud-12', '1LVGqBCRhjpl2OrKaUouh8rEiV3dpBep_', True),
+    ('g-cloud-12', '1LVGqBCRhjpl2OrKaUouh8rEiV3dpBep_', False),
+    ('digital-outcomes-and-specialists-5', '1KqVLn0yHBt3Sd7BJrNwo1mw7C31HCpPB', True),
```
Comment (gidsg) [inline job_definitions/stats_snapshots.yml:14]: Where does the `1KqVLn0yHBt3Sd7BJrNwo1mw7C31HCpPB` value come from out of interest?

Comment (gidsg) [reply job_definitions/stats_snapshots.yml:14]: Oh are they Google Drive folder IDs or something?

**Review by gidsg — APPROVED** [review]
👍 📈

File: job_definitions/stats_snapshots.yml (hunk 1 above)
Comment (lfdebrux) [reply job_definitions/stats_snapshots.yml:14]: It's an existing Google Drive folder, a bunch of them were created a while back, by @benvand I think (see #300).

**Comment by benvand** [comment]
Yup 👍  :)

**Pull request merged**